### ISSUE TITLE
TcpHandler is spamming in Console.Write

### ISSUE THREAD

**Issue by deniszykov** [StackExchange/NetGain#5]
It would be polite for the NetGain do not write logs directly to console.  
For module with so many abstractions, it is strange that that logging is done through the console.

**Comment by mgravell** [comment]
That's fair. Will revisit.

**Comment by mleenhardt** [comment]
Hey @mgravell, thanks for that awesome project! Has there been any progress on this? We're interested in using the lib but having hardcoded Console.Writeline isn't an option for us and we would ideally like to not fork the project just to fix this. #10 seems like a decent alternative while other .net core logging discussion are happening. Would you consider it meanwhile?

**Comment by mgravell** [comment]
Yes; I have some incomplete changes to push on this - I will look at the logging at the same time

**Comment by mgravell** [comment]
K, so I've uploaded https://www.nuget.org/packages/StackExchange.NetGain/1.0.39 which makes `ErrorLog` and `Log` available on the handler or service; they *default* to `Console.Out` or `Console.Error` to be equivalent, but you can either `null` them out or provide your own.

**Comment by mleenhardt** [comment]
Thank you! The updated package currently shows as unlisted and won't by picked up by a nuget update, just making sure that's intended. I'll manually download it for now. 😃

**Comment by mgravell** [comment]
Yes that was intended. I didn't have the energy to change the build server
to get it marked as beta. I'm bad.
